### PR TITLE
option to enable autoscaling in ECS deployment

### PR DESCRIPTION
**Pull request by rpoluri**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -10,6 +10,8 @@ For more information please refer to the main [Apiary](https://github.com/Expedi
 | aws_region | AWS region to use for resources. | string | - | yes |
 | bastion_ssh_key_secret_name | Secret name in AWS Secrets Manager which stores the private key used to log in to bastions. The secret's key should be `private_key` and the value should be stored as a base64 encoded string. Max character limit for a secret's value is 4096. | string | `` | no |
 | cpu | The number of CPU units to reserve for the Waggle Dance container. Valid values can be 256, 512, 1024, 2048 and 4096. Reference: https://docs.aws.amazon.com/AmazonECS/latest/developerguide/task-cpu-memory-error.html | string | `1024` | no |
+| cpu_scale_in_cooldown | cool down time of scale in task by cpu usage | number | 300 | no |
+| cpu_scale_out_cooldown | cool down time of scale out task by cpu usage | number | 120 | no |
```
Comment (barnharts4) [inline README.md:14]: Is this in seconds?

File: ecs.tf [hunk 2]
```diff
@@ -40,3 +52,83 @@ resource "aws_ecs_task_definition" "waggledance" {
...
+  vpc_id      = var.vpc_id
+  tags        = var.tags
+  health_check {
+    protocol = "HTTP"
+    port     = 18000
+    path     = "/actuator/health"
+    //interval = 10
+    //healthy_threshold   = 3
+    //unhealthy_threshold = 3
+    //matcher             = var.healthcheck_matcher
```
Comment (barnharts4) [inline ecs.tf:116]: Are these comments here because you want to enable this at some point?

**Review by barnharts4 — APPROVED** [review]

File: ecs.tf (hunk 2 above)
Comment (rpoluri) [reply ecs.tf]: yes, cleaned up anyway..

File: README.md (hunk 1 above)
Comment (rpoluri) [reply README.md]: yes, updated

**Review by barnharts4 — APPROVED** [review]

**Pull request merged**